### ISSUE TITLE
dsconf should support PAM pass-throught config option: pamModuleIsThreadSafe

### ISSUE THREAD

**Issue by tbordaz** [389ds/389-ds-base#5685]
**Issue Description**
A new configuration option (for the PAM configuration instance) is pamModuleIsThreadSafe (see #5271).
dsconf should support it like

dsconf -D "cn=Directory Manager" ldap://server.example.com plugin pass-through-auth pam-config "Thread safe PAM PTA Config" add --exclude-suffix="cn=config" --id_map_method="RDN ENTRY" --id-attr="customPamUid" --filter="(manager=uid=example_user,ou=people,dc=example,dc=com pamFallback: FALSE" --secure="TRUE" --thread-safe="TRUE" --service="ldapserver"

**Package Version and Platform:**
since #5271 

**Steps to Reproduce**

**Expected results**

**Screenshots**

**Additional context**

**Comment by Firstyear** [comment]
I don't think we should be advetising this in the ui at all, be it the cli or webui. This is a risky option that requires deep knowledge of if the pam module really is thread safe. It should only be set in rare circumstances.

**Comment by tbordaz** [comment]
> I don't think we should be advetising this in the ui at all, be it the cli or webui. This is a risky option that requires deep knowledge of if the pam module really is thread safe. It should only be set in rare circumstances.

I agree with you. Risk of such PAM tuning has been a long standing concern and advertising it is likely wrong. I would wait a bit for others feedback to decide what to do with this ticket.
